doc/source/multivisit.rst

```diff
@@ -3,4 +3,435 @@
+solving a number of key problems; namely,
+
+  * aligning all the images to the same coordinate system
+  * define the size on the sky of each mosaic
+  * defining what exposures should go into each mosaic
+
+MVM processing implemented as part of the Hubble Advanced Products (HAP) pipeline generates new products based on
+solutions implemented for these critical issues.
+
+
+Alignment
+==========
+Generating MVM products relies entirely on the definition of the WCS of each input image in order to define where each
+exposure will land in the output mosaic.  Errors in an input exposure's WCS will lead to misalignment of overlapping
+images resulting in smeared sources or even multiple images of the overlapping sources in the output mosaic.  Errors
+in the WCS primarily stem from images being aligned to different astrometric catalogs.  For example,
+one exposure may be aligned successfully to the GAIADR2 catalog, yet another overlapping exposure can only be aligned
+to the 2MASS catalog due to the lack of GAIA sources in the offset exposure.
+
+MVM processing relies entirely on the alignment that can be performed on the exposures during standard pipeline processing
+and subsequently as part of the Single-Visit Mosaic (SVM) processing.  This allows each visit to be aligned to the most
+accurate catalog available while taking into account the proper motions of the astrometric sources in the field based on
+the date the exposures were taken in the visit.
+
+By default, the most current GAIA catalog will
+be used as the default astrometric catalog for aligning all exposures which typically results in images that are aligned
+well enough to avoid serious misalignment artifacts in the regions of overlap. Unfortunately, due to the nature of some
+of the fields observed by HST, not all exposures can be aligned to a GAIA-based
+catalog using GAIA astrometric sources.  As a result, the sources used for alignment will have much larger errors on the
+sky compared to the errors of GAIA astrometric sources which can lead to offsets from the GAIA coordinate system of a
+significant fraction of an HST pixel (or even multiple HST pixels).  This will lead to artifacts in MVM products where
+such exposures overlap other exposures aligned to other catalogs, especially GAIA-based catalogs.
+
+
+Defining the Mosaic on the Sky
+==============================
+MVM products need to be defined in order to understand what input exposures will contribute to each mosaic.  The solution
+implemented for HAP MVM products relies on tesselation of the entire sky using the same basic tiles defined by the
+PanSTARRS project as described at `PanSTARRS Sky tessellation patterns
+<https://outerspace.stsci.edu/display/PANSTARRS/PS1+Sky+tessellation+patterns>`_.
+
+.. figure:: images/figure_aitoff.png
+  :figwidth: 90%
+  :alt: Aitoff plot of all 2,009 PS1 projection cells for the 3PI survey.
+
```

Comment:
What is the numbering convention for the projection cells?

Reply:
Changed in 241a672